README.md

```diff
@@ -1,48 +1,73 @@
-Role Name
+Pacifica
 =========
 
-A brief description of the role goes here.
+This Ansible role defines configuration for deploying Pacifica services in an
+Ansible managed infrastructure.
 
 Requirements
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the boto package is required.
+There are no requirements for this role currently. However, the Pacifica
+services do depend on other services depending on configuration. So, please
+be aware if you configure a service to use PostgreSQL you should configure
+PostgreSQL prior to includeing this role.
 
 Role Variables
 --------------
 
-A description of the settable variables for this role should go here, including
-any variables that are in defaults/main.yml, vars/main.yml, and any variables
-that can/should be set via parameters to the role. Any variables that are read
-from other roles and/or the global scope (ie. hostvars, group vars, etc.) should
-be mentioned here as well.
+The role variables are to install Python on the different distributions. This
+can be overridden by setting the `setup_packages` and `python_packages` to
+a list of custom packages in a `default.yml` in a consuming role.
```

Comment:
Consider adding the ability to not install python via this role in some cases, such as:
```
-a list of custom packages in a `default.yml` in a consuming role.
+a list of custom packages in a `default.yml` in a consuming role. Alternatively, if the consuming playbook is installing python by other means, you can set `external_python` to any value and the python installation task will be skipped.
```

Reply:
Done